### ISSUE TITLE
Adding import from seed phrase option

### ISSUE THREAD

**Issue by bethatguyad** [FuelLabs/fuels-wallet#1117]
### Motivation

Import wallet account from seed phrase is a widely used feature, just like other wallet.other wallets

**Comment by helciofranco** [comment]
You can click on `Import seed phrase`.

<img width="494" alt="Screenshot 2024-02-28 at 13 47 55" src="https://github.com/FuelLabs/fuels-wallet/assets/7074983/b58d382c-7c9c-4f96-a08b-3c4c9ae5ef2b">

**Comment by bethatguyad** [comment]
No, I mean when you have multiple accounts, your illustration is imported from the seed phrase when setting up,  I mean after you set up your wallet.

**Comment by helciofranco** [comment]
@bethatguyad i see. Can you provide some examples, please? You mean importing with private keys?

**Comment by bethatguyad** [comment]
e.g., okx wallet: 
![image](https://github.com/FuelLabs/fuels-wallet/assets/160697994/99de4dfe-d2ca-4737-8cfe-3274ad78b69a)
when you click `import wallet`

![image](https://github.com/FuelLabs/fuels-wallet/assets/160697994/95ece9ba-abf2-4cd9-aba3-f23c30638155)
they have two options, one for **import from private key** and one for `import from seed phrase`

**Comment by prince596** [comment]
Nice observation. Most wallets have that option so i think it's worth looking into

**Comment by luz33c** [comment]
I need 24 words mnemonic phrase

**Comment by LuizAsFight** [comment]
@bethatguyad this is not possible cuz as we use BIP44 , when you import the seed phrase, thats the "root" account of your wallet. When adding accounts the new ones will be always the same account as they are derived from root. If we import from seed phrase it would be only the first account, and no derivation would be allowed.

I understand the issue purpose, just mentioning its a bigger effort than just adding another import method.

We will work on prioritization for this issue but no ETA yet
